addons/mail/static/src/core/web/chatter.js

```diff
@@ -271,21 +263,53 @@ export class Chatter extends Component {
     /**
      * @param {number} threadId
      * @param {['activities'|'followers'|'attachments'|'messages'|'suggestedRecipients']} requestList
+     * @param {boolean} refresh
+     * Fetch data for the thread according to the request list.
      */
     load(
         threadId = this.props.threadId,
-        requestList = ["followers", "attachments", "messages", "suggestedRecipients"]
+        requestList = ["followers", "attachments", "messages", "suggestedRecipients"],
+        refresh = true
     ) {
         const { threadModel } = this.props;
-        this.state.thread = this.threadService.getThread(threadModel, threadId);
-        this.scrollPosition.model = this.state.thread?.scrollPosition;
+        const thread = this.threadService.getThread(threadModel, threadId);
+        if (refresh) {
+            this.refresh(threadModel, threadId);
+        }
```

Comment:
"load" should not do anything else than `fetchData`, we should remove this param and call that method outside when necessary

Reply:
yes, the reason i was doing this was `load()` is being overwritten in some other places. But I agree with  you and i will split it out and add extra codes in the places where being overwritten